src/app/components/homepage/ATLAS job list.code-workspace

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
what is this file?

Reply:
I deleted it. it was a file I saved wrong